packages/shorebird_cli/lib/src/commands/patch/patch_aar_command.dart

```diff
@@ -88,13 +99,17 @@ of the Android app that is using this module.''',
     }
 
     final force = results['force'] == true;
-    final dryRun = results['dry-run'] == true;
-
-    if (force && dryRun) {
-      logger.err('Cannot use both --force and --dry-run.');
+    if (force) {
+      logger
+        ..err(PatchCommand.forceDeprecationErrorMessage)
```

Comment:
Isn't this technically a breaking change and not a deprecation if we're changing the behavior of `--force`? Shouldn't we not exit to maintain backward compatibility here?

Reply:
Discussed offline. Resolving.